costmap_cspace/src/largemap_to_map.cpp

```diff
@@ -64,22 +61,20 @@ class LargeMapToMapNode
 
 public:
   LargeMapToMapNode()
-    : pnh_("~")
-    , nh_()
+    : Node("largemap_to_map")
+    , tfbuf_(this->get_clock())
     , tfl_(tfbuf_)
   {
-    neonavigation_common::compat::checkCompatMode();
-    pnh_.param("robot_frame", robot_frame_, std::string("base_link"));
+    // neonavigation_common::compat::checkCompatMode();
+    robot_frame_ = this->declare_parameter("robot_frame", std::string("base_link"));
 
-    pub_map_ = neonavigation_common::compat::advertise<nav_msgs::OccupancyGrid>(
-        nh_, "map_local",
-        pnh_, "map", 1, true);
-    sub_largemap_ = nh_.subscribe("map", 2, &LargeMapToMapNode::cbLargeMap, this);
+    pub_map_ = this->create_publisher<nav_msgs::msg::OccupancyGrid>(
+        "~/map", rclcpp::QoS(1).transient_local());
```

Comment:
"~/map" is backward style. Use "map_local".
```suggestion
        "map_local", rclcpp::QoS(1).transient_local());
```